plugin/PDFDocumentPage.qml

```diff
@@ -189,9 +375,182 @@ DocumentPage {
+                     && contextMenuText.annotation !== null
+            //% "Edit"
+            text: qsTrId("sailfish-office-me-pdf-txt-anno-edit")
+            onClicked: pdfDocument.edit(contextMenuText.annotation)
+        }
+        MenuItem {
+            visible: contextMenuText.annotation !== undefined
+                     && contextMenuText.annotation !== null
+            //% "Delete"
+            text: qsTrId("sailfish-office-me-pdf-txt-anno-clear")
+            onClicked: contextMenuText.annotation.remove()
+        }
+        Component {
+            id: textComponent
+            PDF.TextAnnotation { }
+        }
+    }
+
+    ContextMenu {
+        id: contextMenuHighlight
+        property variant annotation
+
+        InfoLabel {
+            id: infoContents
+            visible: infoContents.text != ""
+            width: parent.width
+            anchors.topMargin: Theme.paddingSmall
+            height: implicitHeight + 2 * Theme.paddingSmall
+            font.pixelSize: Theme.fontSizeSmall
+            wrapMode: Text.Wrap
+            elide: Text.ElideRight
+            maximumLineCount: 2
+            color: Theme.highlightColor
+            opacity: .6
+            text: {
+                if (contextMenuHighlight.annotation !== null
+                    && contextMenuHighlight.annotation !== undefined
+                    && contextMenuHighlight.annotation.contents != "") {
+                    return (contextMenuHighlight.annotation.author != ""
+                            ? "(" + contextMenuHighlight.annotation.author + ") " : "")
+                    + contextMenuHighlight.annotation.contents
+                } else {
+                    return ""
+                }
+            }
+        }
+        Row {
+            height: Theme.itemSizeExtraSmall
+            Repeater {
+                id: colors
+                model: ["#db431c", "#ffff00", "#8afa72", "#00ffff",
+                "#3828f9", "#a328c7", "#ffffff", "#989898",
+                "#000000"]
+                delegate: Rectangle {
+                    width: contextMenuHighlight.width / colors.model.length
+                    height: parent.height
+                    color: modelData
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: {
+                            contextMenuHighlight.hide()
+                            contextMenuHighlight.annotation.color = color
+                            highlightColorConfig.value = modelData
+                        }
+                    }
+                }
+            }
+        }
+        Row {
+            height: Theme.itemSizeExtraSmall
+            Repeater {
+                id: styles
+                model: [{"style": PDF.HighlightAnnotation.Highlight,
+                         "label": "abc"},
+                        {"style": PDF.HighlightAnnotation.Squiggly,
+                         "label": "a&#816;b&#816;c&#816;"},
```

Comment:
Hmh, wonder if &#x330; would be easier to check what the character is, unicode more commonly presented in hex.